clang/test/CodeGen/ifunc-win.c

```diff
@@ -0,0 +1,65 @@
+// RUN: %clang_cc1 -triple aarch64-pc-windows-msvc -emit-llvm -o - %s | FileCheck %s
```

Comment:
Is this test file specific to AArch64? If so is it worth adding an 'aarch64-' prefix to it?

Reply:
So far, but other targets could be added here later without too much change to the test.